tock/tock/templates/base.html

```diff
@@ -12,7 +12,7 @@
     <link rel="stylesheet" href="{% static 'css/vendor/jquery-ui.min.css' %}">
     <link rel="stylesheet" href="{% static 'js/chosen/chosen.min.css' %}">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js" ></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" ></script>
```

Comment:
Another future improvement would be to serve all these together as one file. Also, we generally try and stay away from cdns at 18f, mainly due to security. Is it possible to move the cdn files to the local repo?

Reply:
Ah good call - I'll pull them in directly!